dist/connection.d.ts

```diff
@@ -0,0 +1,15 @@
+import { IConnection, IOptions } from "./util/types/connectionTypes";
+export declare class Connection {
```

Comment:
add one line break

Reply:
@VikasPatel these are declaration files created via script do you still want me to add the line break